chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1026,6 +1026,14 @@ impl PeerManagerActor {
                     self.handle_msg_network_requests(msg, ctx),
                 )
             }
+            PeerManagerMessageRequest::AdvertiseTier1Proxies => {
+                let state = self.state.clone();
+                let clock = self.clock.clone();
+                ctx.spawn(wrap_future(async move {
+                    state.tier1_advertise_proxies(&clock).await;
+                }));
+                PeerManagerMessageResponse::AdvertiseTier1Proxies
+            }
```

Comment:
What about validators that are not in tier1?

Reply:
Just for my context, what are the proxies?

Reply:
[`ValidatorProxies`](https://github.com/near/nearcore/blob/master/chain/network/src/config.rs#L32-L50) - from my understanding proxies allow a validator to have several IP addresses that it listen for incoming traffic.
cc @saketh-are 

[`tier1_advertise_proxies()`](https://github.com/near/nearcore/blob/master/chain/network/src/peer_manager/network_state/tier1.rs#L80-L82) calls [`set_local()`](https://github.com/near/nearcore/blob/master/chain/network/src/accounts_data/mod.rs#L277) that is required to be called after [`set_keys()`](https://github.com/near/nearcore/blob/master/chain/network/src/accounts_data/mod.rs#L206-L212) and it does [tier2 broadcast](https://github.com/near/nearcore/blob/master/chain/network/src/peer_manager/network_state/tier1.rs#L198.) in the end.

There also exists [`tier1_request_full_sync()`](https://github.com/near/nearcore/blob/master/chain/network/src/peer_manager/network_state/tier1.rs#L69-L71) function that
> should be called whenever the accounts_data.keys changes

but it is a subset of `tier1_advertise_proxies()`, it does not call `set_local()` and it is not enough.